src/Compilers/Core/MSBuildTask/GenerateMSBuildAnalyzerConfig.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.CodeAnalysis.BuildTasks
```

Comment:
>namespace [](start = 0, length = 9)

nit: #nullable enable?

Reply:
Nullable is enabled at the project level for this assembly. Stylistically, are we still putting in source?

---
In reply to: [419572135](https://github.com/dotnet/roslyn/pull/43617#discussion_r419572135) [](ancestors = 419572135)

Reply:
Interesting, didn't realize we'd done that.

Reply:
Project-level sounds good to me. Thanks